pandas/core/groupby/groupby.py

```diff
@@ -3228,15 +3231,17 @@ def ngroup(self, ascending: bool = True):
 
         Examples
         --------
-        >>> df = pd.DataFrame({"A": list("aaabba")})
+        >>> df = pd.DataFrame()
```

Comment:
Agree with Matt, also I think the examples can be simplified. I think having just a single column with the `None` value is enough, no need to show without the `None` first, and with it later.

Also, unrelated to your changes, but I'd remove `df.groupby(["A", [1,1,2,3,2,1]]).ngroup()`. This is a very specific case, not directly related to `ngroup`, and grouping with provided values is not even shown in the examples of `groupby`, so it makes less sense to show it here. I think it mostly adds confusion to this docstring.

And also, I think it's better for users if the examples are a bit more meaningful than random letters. I think using something like `df = pd.DataFrame({'color': ['red', 'red', 'green', None, 'red', 'green']})` (or with animals, jobs, whatever... would make the example a bit easier to understand. Just as an idea, if you want to keep this PR only for the example you're trying to illustrate, that's surely fine.

Reply:
I agree with all of these suggestions. fixup PR incoming.